core/templates/pages/exploration-editor-page/translation-tab/modal-templates/add-audio-translation-modal.component.html

```diff
@@ -17,10 +17,6 @@ <h3 tabindex="0">Add Voiceover</h3>
                                        (fileChange)="updateUploadedFile($event)"
                                        (fileClear)="clearUploadedFile()">
             </oppia-audio-file-uploader>
-            <div *ngIf="isAudioAvailable"
-                 class="oppia-updated-audio-file-upload-message" tabindex="0">
-              <strong>Note:</strong> Uploading a new audio file will replace the existing audio file.
-            </div>
```

Comment:
Just to check, reupload will no longer replace the existing file?

Reply:
Ah, this should be present for old feature (without enabling the feature flag). Removed this change. Thank you